DOCSP-37869 Final Cleanup (#84)
mongodb/docs-pymongo: source/includes/troubleshooting/connection-targets.rst

Server Reports Wire Version X, PyMongo Requires Y
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> If you attempt to connect to a MongoDB Server v3.4 or earlier,
> {+driver-short+} throws the following error:
> 
> .. code-block:: 
> 
>    pymongo.errors.ConfigurationError: Server at localhost:27017 reports wire version 5, but this version of PyMongo requires at least 6 (MongoDB 3.6).
> 
> This occurs when the driver version is too new for the server it's connecting to.
> To resolve this issue, upgrade your database to a v3.6 or later, or downgrade to
> PyMongo v3.x, which supports MongoDB Server v2.6 or later.
> 

Reviewer: S:

```suggestion
If you try to connect to MongoDB Server v3.4 or earlier,
{+driver-short+} might throw the following error:

.. code-block:: 

   pymongo.errors.ConfigurationError: Server at localhost:27017 reports wire version 5, but this version of PyMongo requires at least 6 (MongoDB 3.6).

This occurs when the driver version is too new for the server it's connecting to.
To resolve this issue, upgrade your MongoDB deployment to v3.6 or later, or downgrade to
{+driver-short+} v3.x, which supports MongoDB Server v2.6 and later.

```
